inc/Engine/Optimization/DelayJS/Admin/Settings.php

```diff
@@ -193,4 +211,21 @@ public static function exclusion_list_has_default(): bool {
 		return true;
 	}
 
+	/**
+	 * Refresh exclusions option based on selected items option.
+	 *
+	 * @return void
+	 */
+	public function refresh_exclusions_option() {
+		$current_selected_exclusions_selected_exclusion = get_rocket_option( 'delay_js_exclusions_selected_exclusions', '' );
```

Comment:
is there a reason to use `get_rocket_option()` instead of the `$options` instance in the class?